saleor/app/types.py

```diff
@@ -27,7 +39,40 @@ class AppExtensionMount:
     ORDER_OVERVIEW_CREATE = "order_overview_create"
     ORDER_OVERVIEW_MORE_ACTIONS = "order_overview_more_actions"
 
+    DRAFT_ORDER_DETAILS_MORE_ACTIONS = "draft_order_details_more_actions"
+    DRAFT_ORDER_OVERVIEW_CREATE = "draft_order_overview_create"
+    DRAFT_ORDER_OVERVIEW_MORE_ACTIONS = "draft_order_overview_more_actions"
+
+    PROMOTIONS_DETAILS_MORE_ACTIONS = "promotions_details_more_actions"
```

Comment:
I'm wondering if having new UI names (promotions etc) is better than using old ones (one we use in `NAVIGATION_DISCOUNTS` (discounts etc) 🤔